bucket/retdec.json

```diff
@@ -0,0 +1,60 @@
+{
+    "version": "4.0",
+    "description": "A retargetable machine-code decompiler based on LLVM.",
+    "homepage": "https://github.com/avast/retdec",
+    "license": "MIT",
+    "notes": [
+        "The python executables were replaced with *.cmd by the shim creation.",
+        "You could install the following programs:",
+        " - UPX (Optional: if you want to use UPX unpacker in the preprocessing stage);",
+        " - Graphviz (Optional: if you want to generate call or control flow graphs)."
+    ],
+    "suggest": {
+        "Python 3.4+": "python",
+        "Visual C++ Redistributable": "extras/vcredist2022"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/avast/retdec/releases/download/v4.0/retdec-v4.0-windows-64b.7z",
+            "hash": "5fc667da3af3530cf3d393f96627987c3e46da62f4b426e78cdcc25a79416a31"
+        },
+        "32bit": {
+            "url": "https://github.com/avast/retdec/releases/download/v4.0/retdec-v4.0-windows-32b.7z",
+            "hash": "c9b6fc91007c2158209ac730d981a99006a960a27e3b0f54e5618872f34e1afe"
+        }
+    },
+    "extract_dir": "retdec",
+    "bin": [
+        "bin\\retdec-archive-decompiler.py",
+        "bin\\retdec-ar-extractor.exe",
+        "bin\\retdec-bin2llvmir.exe",
+        "bin\\retdec-bin2pat.exe",
+        "bin\\retdec-config.exe",
+        "bin\\retdec-config.py",

```

Comment:
```suggestion
```

exe for this already exists

Reply:
I believe this is fine because the script seems to set environment variables for other app python scripts